src/Session.cpp

```diff
@@ -1449,8 +1464,12 @@ bool CSession::SeekChapter(int ch)
     m_adaptiveTree->next_period_ = m_adaptiveTree->periods_[ch];
     for (auto& stream : m_streams)
     {
-      if (stream->GetReader())
-        stream->GetReader()->Reset(true);
+      ISampleReader* sr{stream->GetReader()};
```

Comment:
Better to use `ISampleReader* sr = stream->GetReader();` No need to a different assignment operator that is not often used.

Reply:
i started use brackets `{}` since enen has required it, as per xbmc rules documentation
if there are different use cases it would be better to explain more in the documentation

Reply:
I am happy to abandon `{}` but we need to clarify its uses so as not to create diatribes between developers

Reply:
Fair point. Let me check the docs. They may have changed.

Reply:
Ha. Well you learn something new every day. This comment is just noise please ignore.